core/src/components/textarea/textarea.tsx

```diff
@@ -169,28 +169,22 @@ export class Textarea implements ComponentInterface {
    */
   @Event() ionFocus!: EventEmitter<void>;
 
-  /**
-   * Emitted when the input has been created.
-   * @internal
-   */
-  @Event() ionInputDidLoad!: EventEmitter<void>;
-
-  /**
-   * Emitted when the input has been removed.
-   * @internal
-   */
-  @Event() ionInputDidUnload!: EventEmitter<void>;
-
-  componentWillLoad() {
+  connectedCallback() {
     this.emitStyle();
+    this.debounceChanged();
+    document.dispatchEvent(new CustomEvent('ionInputDidLoad', {
```

Comment:
Can this get changed to use `this.el` instead? Stencil would be able to run with `document` just fine, but I'd be worried about other systems choking on it.